Misc/NEWS.d/next/Library/2018-01-15-12-53-13.bpo-32521.IxX4Ba.rst

```diff
@@ -0,0 +1 @@
+The nismodule is now compatible with new libnsl and headers location.
```

Comment:
Again, would you mind to mention the glibc version here?

Reply:
That would be version 2.26. Relevant changes 
[here](https://sourceware.org/git/?p=glibc.git;a=blob;f=NEWS;h=79736dbee8c1721f9de694a488d086a93ac51dbd;hb=HEAD#l306) and [here](https://sourceware.org/git/?p=glibc.git;a=blob;f=NEWS;h=79736dbee8c1721f9de694a488d086a93ac51dbd;hb=HEAD#l311)

Reply:
"nismodule" => "nis module"?